build.xml

```diff
@@ -8,50 +8,61 @@
 
     <!-- Target: install-all -->
     <target name="install-all" description="Install all three Drupal sites in the Federated Search Demo">
+        <phingcall target="solr-clear" />
         <phingcall target="install-d7" />
 
         <phingcall target="install-d8">
             <property name="drush.root" value="web/d8/docroot" override="true" />
+            <property name="site.name" value="Federated Search Drupal 8" override="true" />
```

Comment:
Is this not set in config? (if not, that's kind of annoying! but ok :)

Reply:
It is set in config but not respected on build. We could add another `drush cim` step, but that seems really odd.